panos_upgrade_assurance/exceptions.py

```diff
@@ -37,6 +37,12 @@ class CommandRunFailedException(FirewallProxyException):
     pass
 
 
+class GetXpathConfigFailedException(FirewallProxyException):
```

Comment:
Not necessarily but maybe we can avoid using both Failed and Exception like `GetXpathConfigException`

Reply:
yeah, we can change it, I was just following the exception format for the *twin* method: `op_parser`. There the exception is: `CommandRunFailedException`

Reply:
Ok then let's keep it.